snorkel/classification/snorkel_classifier.py

```diff
@@ -250,11 +250,13 @@ def calculate_loss(
         loss_dict = dict()
         count_dict = dict()
 
-        task_names = Y_dict.keys()
+        # Collect the available tasks for this module
+        task_names = [name for name in Y_dict.keys() if name in self.task_flows]
```

Comment:
helper class method? seems like we're doing this a lot

Reply:
good idea